HF prep: generate a keypair restructure, plus node operator metadata and light edits (#605)
o1-labs/docs2: docs/node-operators/staking-and-snarking.mdx

> As a snark worker, you get to share some of the block reward for each block your compressed transactions make it in to. The block producer is responsible for gathering compressed transactions before including them into a block, and will be incentivized by the protocol to reward snark-workers.

Author: ```suggestion
As a SNARK worker, you get to share some of the block rewards for each block your compressed transactions make it into. The block producer is responsible for gathering compressed transactions before including them in a block and is incentivized by the protocol to reward SNARK workers.
```
